src/cmap/auth/mongodb_oidc.ts

```diff
@@ -11,11 +11,8 @@ import type { Workflow } from './mongodb_oidc/workflow';
  * @experimental
  */
 export interface OIDCMechanismServerStep1 {
-  authorizationEndpoint?: string;
-  tokenEndpoint?: string;
-  deviceAuthorizationEndpoint?: string;
+  issuer?: string;
```

Comment:
```suggestion
  issuer: string;
```

it's not marked optional in the [server IDL](https://github.com/10gen/mongo-enterprise-modules/blob/8cdcc2da55448c3f85f9a28aad958ae11f7dc55f/src/sasl/oidc_parameters.idl#L13-L19) either, and the callback needs this value :)

Reply:
Updated.